src/components/button/index.tsx

```diff
@@ -109,7 +109,7 @@ class Button extends PureComponent<Props, ButtonState> {
     } else if (outline) {
       color = outlineColor || Colors.$textPrimary;
     } else if (this.isIconButton) {
-      color = undefined; // Colors.grey10;
```

Comment:
Do we want to change the (public) screen as well?

Reply:
Yup, I updated the screen :)